src/manifolds/ShapeSpace.jl

```diff
@@ -0,0 +1,120 @@
+
+@doc raw"""
+    ShapeSpace{n,k} <: AbstractDecoratorManifold{ℝ}
+
+Kendall's shape space, defined as quotient of a pre-shape space (equivalent to
+[`ArraySphere`](@ref) represented by n×k matrices) by the action [`ColumnwiseMultiplicationAction`](@ref)
```

Comment:
Yes, we have pre shape space now, it seems you're looking at an outdated version here.

Reply:
Oh yeah, sorry was a little confused still when I wrote this and forgot to send it and it was sent with the projection remark review today.
This is clarified already because we renamed this :)